nautobot_golden_config/jobs.py

```diff
@@ -435,14 +361,12 @@ class Meta:
 
         name = "Deploy Config Plans"
         description = "Deploy config plans to devices."
+        has_sensitive_variables = False
 
     def run(self, data, commit):
         """Run config plan deployment process."""
         self.log_debug("Starting config plan deployment job.")
         config_deployment(self, data, commit)
-        if commit and not self.failed:
-            config_plan_qs = data["config_plan"]
-            config_plan_qs.delete()
```

Comment:
If you don't want them deleted after deployment we should at least change the status as suggested by @mzbroch and add that status to the failure state of the deployment job.

Reply:
I have started to make headway on that for 2.0, but will not be in this PR: https://github.com/nautobot/nornir-nautobot/pull/107/files#diff-8068ef3cf2f413f68812c745a0b4479113d15e36471021d4c7f7f476b6fd54e9R156-R176